packages/v8-deopt-viewer/src/index.js

```diff
@@ -143,15 +149,16 @@ export default async function run(srcFile, options) {
 
 	// @ts-ignore
 	const require = createRequire(import.meta.url);
-	const webAppIndexPath = require.resolve("v8-deopt-webapp");
-	const webAppStylesPath = webAppIndexPath.replace(
-		path.basename(webAppIndexPath),
-		"style.css"
+	const webAppIndexPath = path.join(
+		path.dirname(require.resolve("v8-deopt-webapp")),
+		"dist",
+		"v8-deopt-webapp.umd.js"
 	);
-	await copyFile(
-		webAppIndexPath,
-		path.join(options.out, "v8-deopt-webapp.umd.js")
+	const webAppStylesPath = path.join(
+		path.dirname(webAppIndexPath),
+		"style.css"
 	);
+	await copyFile(webAppIndexPath, path.join(options.out, "v8-deopt-webapp.js"));
```

Comment:
If I understand the error correctly from playing around locally, can we change the name of the script reference in `packages/v8-deopt-viewer/src/template.html` to be `v8-deopt-webapp.umd.js` instead `v8-deopt-webapp.js` instead of adding this additional logic here?

Reply:
That should also mean we no longer need to change `packages/v8-deopt-webapp/pakage.json` or add an empty index.js script.